py/dml/c_backend.py

```diff
@@ -1231,14 +1231,30 @@ def generate_immediate_after_callbacks(device):
 
 def generate_simple_events_control_methods(device):
     start_function_definition(
-        'void _cancel_simple_events(conf_object_t *_obj, _identity_t domain)')
-    out('{\n', postindent = 1)
+        'void _cancel_simple_delay_events(conf_object_t *_obj, '
+        'conf_object_t *_clock, const _identity_t *_domain)')
+    out('{\n', postindent=1)
     out(crep.structtype(device) + ' *_dev UNUSED = ('
         + crep.structtype(device) + '*)_obj;\n')
```

Comment:
this looks unused? Looks like you can move the function into `dmllib.h` if you remove it?

Reply:
Ah, I assumed `crep.get_evclass(key)` needed `_dev` but upon closer thought it doesn't.

You can't have it in `dmllib.h` in either case. No access to `dml.globals.after_delay_infos`.

Reply:
good point